### PR TITLE
Copy public files to dist

### PR DESCRIPTION
**Pull request by stephan281094**
We might need to revisit the way we handle images and stuff, but for now this makes sure everything in public gets copied to the dist folder.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -47,8 +48,11 @@ module.exports = {
       chunkFilename: "styles.css",
     }),
     new HtmlPlugin({
-      template: "./public/index.html",
+      template: "./src/index.html",
```
Comment (trybick) [inline webpack.config.js:51]: I think we still want "./public/index.html" here

**Review by trybick — APPROVED** [review]
👍

**Comment by trybick** [comment]
~~Oops didn't mean to approve, pls see my comment~~

File: webpack.config.js (hunk 1 above)
Comment (stephan281094) [reply webpack.config.js:51]: That's not possible. HtmlPlugin will put the `index.html` in the `dist` folder, and then the CopyPlugin will copy `public/index.html` into the `dist` folder as well, giving an error that it already exists.

Comment (trybick) [reply webpack.config.js:51]: Sorry I missed at first that this PR moves index.html from public into src. Thanks for clarifying that point too.

**Review by trybick — APPROVED** [review]

**Pull request merged**